src/plugins/visualizations/public/embeddable/visualize_embeddable.tsx

```diff
@@ -233,13 +266,22 @@ export class VisualizeEmbeddable extends Embeddable<VisualizeInput, VisualizeOut
 
     this.transferCustomizationsToUiState();
 
+    this.labelNode = document.createElement('div');
+    this.labelNode.className = `panel-label`;
+    domNode.appendChild(this.labelNode);
+
     const div = document.createElement('div');
     div.className = `visualize panel-content panel-content--fullWidth`;
     domNode.appendChild(div);
+
     this.domNode = div;
 
     const expressions = getExpressions();
-    this.handler = new expressions.ExpressionLoader(this.domNode);
+    this.handler = new expressions.ExpressionLoader(this.domNode, '', {
```

Comment:
I think 2nd argument should better be `undefined` instead of `''`. as empty string is more like "empty expression"